keyboards/macrocat/readme.md

```diff
@@ -0,0 +1,27 @@
+# macrocat
+
+![macrocat](https://i.imgur.com/ItLHGBf.jpg)
```

Comment:
```suggestion
![macrocat](https://i.imgur.com/ItLHGBfh.jpg)
```

Reply:
Thanks for your advice, but what does the extra h do?